instagram/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import dummyData from '../../dummy-data';
+import SearchBar from '../SearchBar/SearchBar.js';
+import PostContainer from './PostContainer.js'
+import {WrapperDiv} from '../Styles'
+
+class PostsPage extends Component {
+  constructor (props){
+    super(props);
+    this.state = {
+      dummyData: [],
+      search: '',
+    }
+  }
+
+  componentDidMount() {
+    this.setState({ dummyData: dummyData});
+  }
+
+  handleSearch = event => {
+    this.setState({[event.target.name]: event.target.value})
```

Comment:
an alternative could be to destructor the parameter & the target. 
call the parameter target so the we make use of the es6 object shorthand notation 
({ target }) => {
      const { name, value } = target
      this.setState({
        [name]: value
      })
    }